### PR TITLE
RELEASE v1.5.1-incubating

### PR DESCRIPTION
**Pull request by cfmcgrady**

### CODE REVIEW

**Comment by cfmcgrady** [comment]
@yaooqinn @pan3793 @ulysses-you

**Comment by yaooqinn** [comment]
we can remove all **.md.txt **.rst.txt

**Comment by cfmcgrady** [comment]
> we can remove all **.md.txt **.rst.txt

Done

File: content/docs/r1.5.1-incubating/community/collaborators.html [hunk 1]
```diff
@@ -0,0 +1,274 @@
+
+
+<!DOCTYPE html>
+<html class="writer-html5" lang="en" >
+<head>
+  <meta charset="utf-8" />
+  
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  
+  <title>2. Collaborators &mdash; Kyuubi 1.5.2-SNAPSHOT documentation</title>
```
Comment (yaooqinn) [inline content/docs/r1.5.1-incubating/community/collaborators.html:10]: the version is not right

File: src/release/1.5.1-incubating.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+
+### ChangeList
+
+- [[KYUUBI #1442] Fix encode MySQLHandshakePacket UT](https://github.com/apache/incubator-kyuubi/commit/c85d6406)
+
+
+
```
Comment (yaooqinn) [inline src/release/1.5.1-incubating.md:29]: nit: blanklines

**Pull request merged**